numba_dpex/kernel_api_impl/spirv/target.py

```diff
@@ -119,6 +120,39 @@ class SPIRVDevice(GPU):
 target_registry[SPIRV_TARGET_NAME] = SPIRVDevice
 
 
+class _DIBuilder(debuginfo.DIBuilder):
```

Comment:
May be something specific to numba dpex is a better choice for naming?
```suggestion
class SPIRVDIBuilder(debuginfo.DIBuilder):
```

Reply:
I moved the class into `numba_dpex.core.debuginfo` and renamed it to `DIBuilder`.